### PR TITLE
Fix detection of gevent threading.local monkey-patch

### PR DESCRIPTION
**Pull request by bluetech**
The `_` in `_threading` seems like a mistake and always returns False.
Removing it leads to the detection working properly:

```py
Python 3.7.4 (default, Jul 16 2019, 07:12:58)
[GCC 9.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import gevent.monkey
>>> gevent.monkey.is_object_patched('_threading', 'local')
False
>>> gevent.monkey.is_object_patched('threading', 'local')
False
>>> gevent.monkey.patch_all()
True
>>> gevent.monkey.is_object_patched('_threading', 'local')
False
>>> gevent.monkey.is_object_patched('threading', 'local')
True
```

### CODE REVIEW

**Comment by bluetech** [comment]
We are using gevent and are seeing some data "leaking" between events. I hope this will fix the problem, but I did not verify it.

**Comment by untitaker** [comment]
Thanks!

File: tests/utils/test_contextvars.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import time
+
+import gevent
+
+
+from sentry_sdk.utils import _is_threading_local_monkey_patched
+
+
+def test_gevent_is_patched():
+    gevent.monkey.patch_all()
```
Comment (bluetech) [inline tests/utils/test_contextvars.py]: I wonder, does this not affect all of the subsequent tests in the test run?

Comment (untitaker) [reply tests/utils/test_contextvars.py]: We run each test in its own subprocess because the SDK itself is already so heavy on monkeypatching.

Comment (bluetech) [reply tests/utils/test_contextvars.py]: Ha, nice.

**Pull request merged**